src/sessions/base-session-handler.ts

```diff
@@ -29,6 +29,10 @@ export default abstract class BaseSessionHandler {
 
   constructor (protected sdk: GenesysCloudWebrtcSdk, protected sessionManager: SessionManager) {
     this.sdk.on("sessionEnded", (session, reason) => {
+      if (session.sessionType !== this.sessionType) {
```

Comment:
This fixes the log with the `sessionEnded event` which was triggering for all session handlers, not just the one that handles the ended session.